tests/difference/core/quint_model/ccv_test.qnt

```diff
@@ -0,0 +1,255 @@
+        )
+        not(result.hasError) and
+        result.newState.providerState.chainState.votingPowerHistory == List()
+    }
+
+    // Defines a test state to test the deliverPacketToProvider function against.
+    pure val _DeliverPacketToProvider_TestState =
+        val currentState = GetEmptyProtocolState
+        val sender = "sender"
+        val providerState = currentState.providerState
+        val consumerState = GetEmptyConsumerState
+        // add the consumer to the consumerStates
+        val consumerStates = currentState.consumerStates.put(sender, consumerState)
+        val providerState2 = providerState.with(
+            "consumerStatus", providerState.consumerStatus.put(sender, RUNNING)
+        )
+        val providerState3 = providerState2.with(
+            "outstandingPacketsToConsumer", providerState2.outstandingPacketsToConsumer.put(sender, List({
+                id: 0,
+                validatorSet: Map(),
+                sendingTime: 0
+            }))
+        )
+        currentState.with(
+            "providerState", providerState3
+        ).with(
+            "consumerStates", consumerStates
+        )
+
+    // add a packet on the consumer
+    pure val DeliverPacketToProviderHappyPathTest_testState = _DeliverPacketToProvider_TestState.with(
+        "consumerStates", _DeliverPacketToProvider_TestState.consumerStates.put(
+            "sender", _DeliverPacketToProvider_TestState.consumerStates.get("sender").with(
+                "outstandingPacketsToProvider", List({
+                    id: 0,
+                    sendingTime: 0
+                })
+            )
+        )
+    ).with(
+        // put an entry into sentVscPacket on the provider that corresponds to the packet we put on the consumer
+        "providerState", _DeliverPacketToProvider_TestState.providerState.with(
+            "sentVscPacketsToConsumer", _DeliverPacketToProvider_TestState.providerState.sentVscPacketsToConsumer.put(
+                "sender", List({
+                    id: 0,
+                    validatorSet: _DeliverPacketToProvider_TestState.providerState.chainState.currentValidatorSet,
+                    sendingTime: 0
+                })
+            )
+        )
+    )
+
+    pure val DeliverPacketToProviderHappyPathTest_resultAndTimeout = deliverPacketToProvider(DeliverPacketToProviderHappyPathTest_testState, "sender")
+
+    // test is split to be easier to pinpoint which assertion failed
+    run DidNotTimeOut_DeliverPacketToProviderHappyPathTest =
+    {
+        val result = DeliverPacketToProviderHappyPathTest_resultAndTimeout._1
+        val timeout = DeliverPacketToProviderHappyPathTest_resultAndTimeout._2
+        not(result.hasError) and
+        not(timeout)
+    }
+
+    run StateModificationOK_DeliverPacketToProviderHappyPathTest =
+    {
+        val result = DeliverPacketToProviderHappyPathTest_resultAndTimeout._1
+        val timedOut = DeliverPacketToProviderHappyPathTest_resultAndTimeout._2
```

Comment:
```suggestion
```
Doesn't seem to be used. If used maybe use consistent naming with `timeout` above.